.github/workflows/deploy_reusable.yml

```diff
@@ -117,4 +117,4 @@ jobs:
       uses: azure/CLI@v1
       with:
         inlineScript: |
-            az storage blob upload-batch --account-name cdctistagingdocs --auth-mode key -d '$web' -s ${{ steps.extract.outputs.destination }}
+            az storage blob upload-batch --account-name cdcti${{ inputs.ENVIRONMENT }}docs --overwrite --auth-mode key -d '$web' -s ${{ steps.extract.outputs.destination }}
```

Comment:
we can also try using `az storage blob sync...` if `--overwrite` doesn't do what we expect.  It wasn't clear if that would delete files that should be deleted.